infra/modules/service/main.tf

```
@@ -52,9 +60,37 @@ resource "aws_lb" "alb" {
   # Note that header names cannot contain underscores
   # https://docs.bridgecrew.io/docs/ensure-that-alb-drops-http-headers
   drop_invalid_header_fields = true
-
-  # TODO(https://github.com/navapbc/template-infra/issues/162) Add access logs
   # checkov:skip=CKV_AWS_91:Add access logs in future PR
+  access_logs {
+    bucket  = aws_s3_bucket.load_balancer_logs.id
+    prefix  = "${var.service_name}-lb"
+    enabled = true
+  }
+}
+
+resource "aws_s3_bucket" "load_balancer_logs" {
+  bucket = "${var.service_name}-access-logs${var.s3_suffix}"
+}
+
+resource "aws_s3_bucket_policy" "log_access_bucket_policy" {
+  bucket = aws_s3_bucket.load_balancer_logs.id
+  policy = data.aws_iam_policy_document.log_access_bucket_pol_doc.json
+}
+
+data "aws_iam_policy_document" "log_access_bucket_pol_doc" {
```

naming nit: similar to previous comment, since the resource already mentions that it's a policy document we don't need to put that in the name, and instead we can include more info about the purpose e.g.

```suggestion
data "aws_iam_policy_document" "load_balancer_logs_put_access" {
```

could also be `load_balancer_logs_bucket_put_access` or `lb_logs_bucket_put_access`, or `alb_logs_put_access`, i'm fine with any of those, but will probably want to be consistent between the name for the aws_iam_policy_document and the name for the aws_s3_bucket_policy